packages/nextjs-kit/src/__tests__/snapshotTests/paginator.test.tsx

```diff
@@ -83,66 +57,126 @@ const RenderCurrentItems: React.FC<PaginationItemProps> = ({
 describe(`<Paginator />`, () => {
   it('should render the paginated data', () => {
     const { asFragment } = render(
-      <TestComponentToRender
-        breakpoints={{ start: 6, end: 12, add: 6 }}
-        itemsPerPage={10}
-      />
+      <div className="prose container min-w-full min-h-screen max-w-screen mx-auto">
+        <main className="flex mx-auto flex-col">
+          <section className="mx-auto">
+            <h1 className="my-10">Pagination example</h1>
+            <Paginator
+              data={examplePaginationData}
+              itemsPerPage={10}
+              breakpoints={{ start: 6, end: 12, add: 6 }}
+              routing={true}
+              Component={RenderCurrentItems}
+            />
+          </section>
+        </main>
+      </div>
```

Comment:
I don't think you need the surrounding divs. It's fine to just render the `<Paginator />` here.